src/Plugin/Condition/EntityIsNew.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules\Plugin\Condition\EntityIsOfType.
+ */
+
+namespace Drupal\rules\Plugin\Condition;
+
+use Drupal\rules\Engine\RulesConditionBase;
+
+/**
+ * Provides an 'Entity is new' condition.
+ *
+ * @Condition(
+ *   id = "rules_entity_is_new",
+ *   label = @Translation("Entity is new"),
+ *   category = @Translation("Entity"),
+ *   context = {
+ *     "entity" = @ContextDefinition("entity",
+ *       label = @Translation("Entity"),
+ *       description = @Translation("Specifies the entity for which to evaluate the condition.")
+ *     ),
+ *     "is_new" = @ContextDefinition("boolean",
```

Comment:
That context is not required. See also the D7 implementation which also has only one parameter. If people want the condition to be reversed they can just negate it.